tools/release/sync-stable-blocks.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+/**
+ * Constants
+ */
+const BLOCK_LIBRARY_SRC = 'node_modules/@wordpress/block-library/src/';
+
+const REPOSITORY_ROOT = path.dirname(path.dirname(__dirname));
```

Comment:
Nit: JS coding conventions:
```suggestion
const REPOSITORY_ROOT = path.dirname( path.dirname( __dirname ) );
```